### PR TITLE
Editing Readme

### PR DESCRIPTION
**Pull request by RedPandore**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,90 +1,49 @@
...
+* WARNING : This app use [Open Route Service](https://openrouteservice.org/) API, you also need to get an API key from this service
+* WARNING : This app use [Pole emploi ROMEv1](https://pole-emploi.io/data/api/rome) API, you also need to get an API key from this service
+
+
+## Installation
+if you meet the prerequisites, you can proceed to the installation of the project 
+
+1. Clone the project from [Github](https://github.com/WildCodeSchool/orleans-202103-php-project-jobpermut/)
+2. Go in the project folder
+3. Open with you'r code editor
```
Comment (LucyMerlier) [inline README.md]: your *

File: README.md [hunk 2]
```diff
@@ -1,90 +1,49 @@
...
+4. Run `composer install` to install PHP dependencies
+5. Run `yarn install` to install JS dependencies
+6. Copy the `.env` file and fill all informations (Database, Symfony/Mailer, Open Route Service, Pole Emploi Variable)
+7. Run `symfony console doctrine:database:create` to create database
+8. Run `symfony console doctrine:migration:migrate` to create structure of database
+9. Run `symfony console doctrine:fixtures:load` to load the fixtures in database
+10. Run `yarn encore dev` to build assets
+11. Run `symfony server:start` to launch symfony server
+12. Go to localhost:8000 on you'r browser
```
Comment (LucyMerlier) [inline README.md]: your *

File: README.md [hunk 3]
```diff
@@ -1,90 +1,49 @@
...
+* [Yarn 1.*](https://yarnpkg.com/) (check by running yarn -v in your console)
+* [MySQL 8.0.*](https://www.mysql.com/fr/) (check by running mysql --version in your console)
+* [Git 2.*](https://git-scm.com/) (check by running git --version in your console)
+* You will also need a test SMTP connection, which you can configure using tools like Mailtrap
+* WARNING : This app use [Open Route Service](https://openrouteservice.org/) API, you also need to get an API key from this service
+* WARNING : This app use [Pole emploi ROMEv1](https://pole-emploi.io/data/api/rome) API, you also need to get an API key from this service
+
+
+## Installation
+if you meet the prerequisites, you can proceed to the installation of the project
```
Comment (LucyMerlier) [inline README.md]: If * => with a capital "I"

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
this readme is oddly familiar :stuck_out_tongue_winking_eye:

**Review by bouboumael — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**